app/styles/app.css

```diff
@@ -0,0 +1,440 @@
+  font-size: 20px;
+  text-align: left;
+  grid-column-start: 1;
+}
+.cover.page .grid .cover-tomster {
+  font-size: 14px;
+  grid-column-start: 2;
+  width: 150px;
+  position: relative;
+}
+.cover.page .grid .cover-tomster .top {
+  width: 75%;
+  filter: drop-shadow(3px 3px 3px #444);
+  position: absolute;
+  z-index: 5;
+  top: 7px;
+  left: 12px;
+}
+.cover.page .grid .cover-tomster .top:hover {
+  -webkit-animation: jello-horizontal 0.9s both;
+  animation: jello-horizontal 0.9s both;
+}
+.cover.page .grid .cover-tomster .bottom {
+  width: 90%;
+  -webkit-animation: rotate-center 20s linear infinite both;
+  animation: rotate-center 20s linear infinite both;
+}
+.cover.page .cover-logo img {
+  width: 70%;
+}
+.cover.page .subtitle.lower {
+  margin-top: -10px;
+}
+.cover.page .subtitle.upper {
+  margin-bottom: -5px;
+}
+
+.basic .grid {
+  display: grid;
+  grid-template-columns: 2;
+}
+.basic .grid .left {
+  grid-column-start: 1;
+  align-self: center;
+}
+.basic .grid .right {
+  padding-top: 20px;
+  grid-column-start: 2;
+}
+.basic .grid img {
+  width: 100%;
+}
+
+.basic.data .grid {
+  grid-template-columns: 60% 40%;
+}
+
+.basic.data .right,
+.basic.data .left {
+  padding-top: 0;
+}
+
+.basic.data li {
+  font-size: 18px;
+}
+
+.img-battery {
+  transform: rotate(-25deg);
+}
+
+.img-button:hover {
+  -webkit-animation: heartbeat 1.5s ease-in-out both;
+  animation: heartbeat 1.5s ease-in-out both;
+}
+
+.img-container-battery:hover {
+  -webkit-animation: vibrate-1 0.3s 4 both;
+  animation: vibrate-1 0.3s 4 both;
+}
+
+.img-container-icecream:hover {
+  -webkit-animation: jello-vertical 1.2s both;
+  animation: jello-vertical 1.2s both;
+}
+
+.img-full-width {
+  width: 100%;
+  margin: 0 auto;
+  display: block;
+}
+
+.img-half-width {
+  width: 50%;
+  margin: 0 auto;
+  display: block;
+}
+
+a {
+  color: #e04e39;
+  text-decoration: none;
+}
+
+/* ----------------------------------------------
+ * Generated by Animista on 2018-4-1 22:20:13
+ * w: http://animista.net, t: @cssanimista
+ * ---------------------------------------------- */
+/**
+ * ----------------------------------------
+ * animation rotate-center
+ * ----------------------------------------
+ */
+@-webkit-keyframes rotate-center {
+  0% {
+    -webkit-transform: rotate(0);
+    transform: rotate(0);
+  }
+  100% {
+    -webkit-transform: rotate(360deg);
+    transform: rotate(360deg);
+  }
+}
+
+@keyframes rotate-center {
+  0% {
+    -webkit-transform: rotate(0);
+    transform: rotate(0);
+  }
+  100% {
+    -webkit-transform: rotate(360deg);
+    transform: rotate(360deg);
+  }
+}
+
+/* ----------------------------------------------
+ * Generated by Animista on 2018-4-1 22:29:1
+ * w: http://animista.net, t: @cssanimista
+ * ---------------------------------------------- */
+/**
+ * ----------------------------------------
+ * animation jello-horizontal
+ * ----------------------------------------
+ */
+@-webkit-keyframes jello-horizontal {
```

Comment:
**thought**

I think these `@-webkit-keyframes` definitions are duplicates of those in `animations.css`. I'm not sure why we had had an import statement in the beginning.

In a separate issue and pull request, I think we could look into removing lines 176-440 (partly or entirely).